### ISSUE TITLE
kcl 0.11.0 doesn't work on Amazon Linux 2 anymore (glibc)

### ISSUE THREAD

**Issue by joschi** [twmb/kcl#28]
kcl 0.11.0 doesn't work on Amazon Linux 2 anymore due to incompatibility with the version of glibc on that system.

kcl 0.10.0 and earlier are still working on that distribution.

```
$ wget https://github.com/twmb/kcl/releases/download/v0.11.0/kcl_linux_amd64.gz
--2023-03-08 16:47:42--  https://github.com/twmb/kcl/releases/download/v0.11.0/kcl_linux_amd64.gz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
[...]
Saving to: ‘kcl_linux_amd64.gz’

2023-03-08 16:47:43 (111 MB/s) - ‘kcl_linux_amd64.gz’ saved [12819405/12819405]

$ gunzip kcl_linux_amd64.gz 
$ chmod +x kcl_linux_amd64 
$ ./kcl_linux_amd64 
./kcl_linux_amd64: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./kcl_linux_amd64)
./kcl_linux_amd64: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./kcl_linux_amd64)
$ rpm -q glibc
glibc-2.26-62.amzn2.x86_64
```

**Comment by twmb** [comment]
Should be easily resolveable as of Go 1.20 by switching to CGO_ENABLED=0

**Comment by twmb** [comment]
Released v0.12.0 with CGO_ENABLED=0, please lmk if that does not fix the issue you're running into.
